sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts

```diff
@@ -72,11 +72,14 @@ class GapiManager {
     /**
      * Gets the list of BigQuery projects, returns a Promise.
      */
-    public static listProjects():
+    public static listProjects(pageToken?: string):
         Promise<gapi.client.HttpRequestFulfilled<gapi.client.bigquery.ListProjectsResponse>> {
       const request = {
         maxResults: 1000,
```

Comment:
Add pageToken here in the request definition directly to be consistent with dataset and table requests below?

Reply:
Done.